src/utils/traffic-mesh.js

```diff
@@ -64,13 +77,100 @@ const startForwardProxy = async ({ port, frameworkPort, functionsPort, publishDi
   }
 }
 
+const forwardMessagesToLog = ({ log, subprocess }) => {
+  const { promise: firstBundleReady, reject: firstBundleReject, resolve: firstBundleResolve } = createDeferred()
+
+  let currentId = null
+  let spinner = null
+
+  const reset = () => {
+    currentId = null
+    spinner = null
+  }
+
+  const forwarder = rl.createInterface({
+    input: subprocess.stderr,
+  })
+
+  forwarder
+    .on('line', (line) => {
+      let data
+      try {
+        data = JSON.parse(line.trim())
+      } catch (error) {
+        log(`${NETLIFYDEVERR} Cannot parse log line as JSON: ${line.trim()}${EOL}${EOL}${error}`)
+        return
+      }
+
+      const { error, id, type } = data
+      switch (type) {
+        case 'bundle:start':
+          currentId = id
+          if (!spinner) {
+            spinner = startSpinner({ text: 'Processing request remaps, header rules and Edge Handlers...' })
+          }
+          break
+
+        case 'bundle:success':
+          if (currentId !== id) {
+            return
+          }
+
+          firstBundleResolve()
+          stopSpinner({ spinner, error: false, text: 'Done processing' })
```

Comment:
Not sure what is the correct message here (or vs and), but we should have better granularity since the log is printed with other CLI logs:
![image](https://user-images.githubusercontent.com/26760571/99669554-e700db80-2a77-11eb-8d1e-6877d44f3f24.png)